README.md

```diff
@@ -2,102 +2,103 @@
 
 [![GitHub All Releases](https://img.shields.io/github/downloads/KraXen72/crankshaft/total.svg)](https://github.com/KraXen72/crankshaft/releases/latest) [![Latest release](https://img.shields.io/github/downloads/KraXen72/crankshaft/latest/total)](https://github.com/KraXen72/crankshaft/releases/latest) [![Chat](https://img.shields.io/discord/966300714060116008)](https://discord.gg/ZeVuxG7gQJ) [![Recurring donation via Liberapay](https://img.shields.io/badge/donate-liberapay-%23f6c915?logo=liberapay)](https://liberapay.com/KraXen72) [![One-time donation via ko-fi.com](https://img.shields.io/badge/donate-ko--fi-%23ff5e5b?logo=kofi)](https://ko-fi.com/kraxen72)
 
-> a fast, feature-rich krunker client written in typescript
+> A fast, feature-rich krunker client written in TypeScript
```

Comment:
i didn't use capital letters on purpose, because it kind of evokes too much of an official feeling, but maybe, it's time for that since crankshaft has matured a lot since i initially wrote this and had also gained a bunch of features.
What are your thoughts on small vs capital letters?

Reply:
I usually practice professionalism with my projects, which is why I made changes. It isn't part of the features of the pull request, and I don't mind the ~10 minutes I spent on this being wasted.

Reply:
@Mixaz017 what's you opinion on this? i kinda like lowercase letters but it probably would be better to use proper capitalization, so i'm split.

Reply:
I agree that readme should use proper capitalization (we already do that with release notes), however it should be done in another PR.

Reply:
> I agree that readme should use proper capitalization (we already do that with release notes), however it should be done in another PR.

I agree with this, I'll revert the changes to README in my next commit